laborator/content/operatii-memorie-gdb/demo/pointers/test-pointers.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Author: Gabriel Mocanu <gabi.mocanu98@gmail.com>
+ */
+
+#include <stdio.h>
+
+int main() {
+    
+    int a = 0;
+    int *p;
+    void *pv;
+
+    p = &a;
+    pv = &a;
+
+    printf("Before p = %p\n",p);
```

Comment:
Just a minor thing here
```suggestion
    printf("Before p = %p\n",p);
```